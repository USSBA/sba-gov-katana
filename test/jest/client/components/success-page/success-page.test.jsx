/*global expect*/

/*eslint-disable no-unused-vars*/
import React from 'react';
/*eslint-enable no-unused-vars*/
import SuccessPage from 'client/components/success-page/success-page.jsx';
import renderer from 'react-test-renderer';


test('SuccessPage renders', () => {
    const component = renderer.create(
        <SuccessPage>

        </SuccessPage>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
