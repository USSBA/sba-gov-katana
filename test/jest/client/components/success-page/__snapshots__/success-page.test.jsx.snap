exports[`test SuccessPage renders 1`] = `
<div>
  <div
    className="full-width header">
    <h2
      className="text-center">
      Success
    </h2>
    <div
      className="text-center">
      Next Step Confirm Email
    </div>
  </div>
  <div
    className="container">
    <div
      className="row">
      <div
        className="col-md-6">
        <h3
          className="text-center">
           Dynamic Lender Matches
        </h3>
        <div>
          \"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.\"
        </div>
      </div>
      <div
        className="col-md-6">
        <h3
          className="text-center">
           Dynamic Counseling and Tools
        </h3>
        <div>
          \"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.\"
        </div>
      </div>
    </div>
  </div>
</div>
`;
